api/tests/prescription/campaign/unit/domain/read-models/CampaignToJoin_test.js

```diff
@@ -109,4 +109,46 @@ describe('Unit | Domain | Models | CampaignToJoin', function () {
       expect(campaignToJoin.isRestricted).to.be.false;
     });
   });
+
+  describe('#setReconciliationFields', function () {
+    it('should return null when no reconcilation defined', function () {
+      // given / when
+      const campaignToJoin = domainBuilder.buildCampaignToJoin();
+
+      // then
+      expect(campaignToJoin.reconciliationFields).to.be.null;
+    });
+
+    it('should return given reconciliationFields', function () {
+      // given
+      const campaignToJoin = domainBuilder.buildCampaignToJoin();
+
+      // when
+      const reconciliationFields = Symbol('reconcilationFields');
+      campaignToJoin.setReconciliationFields(reconciliationFields);
+
+      // then
+      expect(campaignToJoin.reconciliationFields).to.be.equal(reconciliationFields);
+    });
+  });
+
+  describe('#isReconciliationRequired', function () {
+    it('should return false when feature not enabled with reconcilationFields', function () {
```

Comment:
**Suggestion :** 
`t('should return false when feature not enabled with reconciliationFields', function () {`